<?xml version="1.0"?>

<launch>
	
  <node name="mocap2odom" pkg="mocap2odom" type="Vicon2odom"/>
 
  <!-- PID Position Controller -->
  <node name="controller" pkg="roscopter" type="controller" output="screen">
	  <!-- <remap from="estimate" to="/vrpn/odom"/> -->
	  <!-- <remap from="estimate" to="/odometry/filtered"/> -->
	  <!-- <remap from="estimate" to="/z_state_estimator/odom"/> -->
	  <remap from="estimate" to="/tf_odom"/>
  </node>

  <rosparam command="load" file="$(find roscopter)/params/gene_F450.yaml"/>


  <!-- Waypoint Manager -->
  <node name="waypoint_manager" pkg="roscopter" type="simple_command_pub.py" output="screen">
	  <!--  <remap from="waypoint" to="high_level_command"/> -->
  </node>

</launch>
